search/includes/classes/class-search.php

```diff
@@ -15,12 +15,16 @@ class Search {
 	private const QUERY_COUNT_TTL = 300; // 5 minutes in seconds.
 	private const MAX_SEARCH_LENGTH = 255;
 	private const DISABLE_POST_META_ALLOW_LIST = array();
+	private const STALE_QUEUE_WAIT_LIMIT = 3600; // 1 hour in seconds
+	private const STALE_QUEUE_ALERT_SLACK_CHAT = '#vip-platform-cantina';
```

Comment:
Do we want this channel, or should we use our other existing ES alerts channel (cc @brandon-m-skinner for thoughts).

Reply:
I managed to test somehow with Nick's help. Still, the channel question is outstanding.

Reply:
Strongly vote for the other channel.